web_ui/ui.go

```diff
@@ -540,6 +655,9 @@ func waitUntilLogin(ctx context.Context) error {
 //
 // You need to mount the static resources for UI in a separate function
 func ConfigureServerWebAPI(ctx context.Context, engine *gin.Engine, egrp *errgroup.Group) error {
+	// start the cache for verified API keys
+	go token.VerifiedKeysCache.Start()
```

Comment:
Connect the go routine to an errgroup object -- that ensures the routine is shut down along with the rest of the server (especially relevant for starting up / shutting down unit tests).